CountItems/masterProcess.js

```diff
@@ -37,6 +51,26 @@ const countMaster = new CountMaster({
     client: new S3UtilsMongoClient(createMongoParams(log)),
 });
 
+const metricServer = new WebServer(8003, log).onRequest((req, res) => monitoring.metricsHandler(
+    () => {
+        if (waitingForPromScraping === true) {
+            countMaster.stop(null, () => process.exit(1));
+        }
+    },
+    req,
+    res,
+));
+metricServer.start();
+
+// Set bucketsCount to 0
+monitoring.bucketsCount.labels('error').set(0);
+monitoring.bucketsCount.labels('success').set(0);
+
+// Set objectsCount to 0
+monitoring.objectsCount.labels('error').set(0);
+monitoring.objectsCount.labels('success').set(0);
+monitoring.objectsCount.labels('skipped').set(0);
```

Comment:
why do you reset the metric to 0?
prometheus series should not be "pre-generated", but only created when the label value is used.

Reply:
Like discussed the approach was to switch to a gauge , this has been dropped